nav2_msgs/msg/BehaviorTreeLog.msg

```diff
@@ -0,0 +1,2 @@
+builtin_interfaces/Time timestamp    # ROS time that this log message was sent.
```

Comment:
This seems like really a use for the diagnostics (https://wiki.ros.org/diagnostics) that was ported to ROS2 already, rather than defining your own new message and interface.